docs/modules/widgets/pages/widgets.adoc

```diff
@@ -0,0 +1,89 @@
+= Widget: Compose Material
+
+This section exposes the custom UI Widget we've already developed so you don't have to reinvent the wheel.
+
+When using Compose, you are probably using Material to handle your theme.
+The downside is that there are two material libraries.
+
+- Material (default 2): `compose.material`
+- Material 3 : `compose.material3`
+
+[[install]]
+== Installing
+
+If you are using `material` and/or `material3`, you can add one or both dependency:
+
+[tabs]
+====
+CuP Plugin::
++
+.build.gradle.kts
+[source, kotlin]
+----
+kotlin {
+    sourceSets.commonMain.dependencies {
+        implementation(cup.widget.material)
+        // and/or implementation(cup.widget.material3)
+    }
+}
+----
+
+Regular Gradle::
++
+.build.gradle.kts
+[source, kotlin, subs="attributes"]
+----
+kotlin {
+    sourceSets.commonMain.dependencies {
+        implementation("net.kodein.cup:cup-widget-material:{version}")
+        // or implementation("net.kodein.cup:cup-widget-material3:{version}")
+    }
+}
+----
+====
+
+[CAUTION]
+====
+Beware that APIs are very much the same, they only differ from their packages, which are:
+- `net.kodein.cup.widget.material`
+- `net.kodein.cup.widget.material3`
+====
+
+[[bullet-points]]
+== Bullet points
+
+Bullet points are a standard way of presenting a list of items in presentations.
+CuP provides a `BulletPoints` composable function that animates each item as it appears:
+
+[source, kotlin]
+----
+BulletPoints {
+    BulletPoint(visible = step >= 1) {
+        Text("The first item.")
+    }
+    BulletPoint(visible = step >= 2) {
+        Text("The second item.")
+    }
+}
+----
+
+[[typography]]
+== Typography downsizing
+
+As we are modifying the density, through CompositionLocals,
+to adapt the content of the presentation Material's typography
+could be oversized in some cases.
+
+To avoid any trouble, we provide a helper function for each Material library,
+thus you can apply a scaling computation to your typography, so they behave properly on your presentation.
```

Comment:
This sentence does not make sense.
Please use the "one sentence per line" Asciidoctor convention (rather than expanding the same sentence over three lines).

As CuP modifies the density to adapt the content of your slides to the presentation screen, the default Material typographies are not adapted to a "scaled up" content and would present big texts (such as `h1` or `displayLarge`) way too large when scaled for your presentation.
To avoid this, CuP provides the `cupScaleDown` helper function (for both Material libraries) which properly scales your typography to be adapted to a CuP presentation.